tests/test_subpixmaxima.py

```diff
@@ -157,22 +127,47 @@ def test_generate_keypoints_batched():
         imgaug_transform=imgaug_transform,
     )
     dl = DataLoader(dataset=dataset, batch_size=10)
-    SubPixMax = SubPixelMaxima(
-        output_shape=(96, 96),
-        output_sigma=torch.tensor(1.25, device=_TORCH_DEVICE),
-        upsample_factor=torch.tensor(2, device=_TORCH_DEVICE),
-        coordinate_scale=torch.tensor(4, device=_TORCH_DEVICE),  # 2 ** 2
-        confidence_scale=torch.tensor(1, device=_TORCH_DEVICE),
-        threshold=None,
-        device=_TORCH_DEVICE,
-    )
-    for batch in dl:
-        gt_keypts = batch["keypoints"].reshape(-1, 17, 2)
-        heatmaps = generate_heatmaps(
-            gt_keypts, height=384, width=384, output_shape=(96, 96)
+    for batch_dict in dl:
```

Comment:
again not sure what we're testing here?

Reply:
agreed, i just auto-replaced everything; ideally we could delete this entire test module eventually